### PR TITLE
feat(starter-kit-blog): Add blog related action and instructions

### PR DESCRIPTION
**Pull request by jeesooxkim**
The Medium blog in progress references a skill that uses both the metaweather and zendesk starter kit. Adding this skill as a resource so that blog readers can follow along if they'd like.

### CODE REVIEW

File: integrations/extensions/starter-kits/metaweather/readme.md [hunk 1]
```diff
@@ -43,9 +43,39 @@ To get started with using the OpenAPI spec, you can upload the `metaweather.adva
       - woeid: 1. OK, I can help with that. What is your ...
       - year: year
       - month: month
+      - date: date
     ```
 If you want to know the Where-On-Earth-ID of a particular location, simply use the first action to query using the location name. The Where-On-Earth-ID of the location will be returned to you.
 
+### Advanced: Blog Spec
```
Comment (amiddavid) [inline integrations/extensions/starter-kits/metaweather/readme.md:50]:
```suggestion
### Even More Advanced: How to use Metaweather in conjunction with another extension
```

File: integrations/extensions/starter-kits/metaweather/readme.md [hunk 2]
```diff
@@ -43,9 +43,39 @@ To get started with using the OpenAPI spec, you can upload the `metaweather.adva
...
+There is [a Medium blog](https://linktoblog) published regarding using this starter kit. The `metaweather-zendesk.blog.actions.json` is a two extension example skill that is highlighted and used in the blog. To configure this spec, you should use the below operations and parameters.
```
Comment (amiddavid) [inline integrations/extensions/starter-kits/metaweather/readme.md:51]: I'm having a problem parsing: "... published regarding using this starter kit"

File: integrations/extensions/starter-kits/metaweather/readme.md (hunk 1 above)
Comment (amiddavid) [inline integrations/extensions/starter-kits/metaweather/readme.md:50]: I would move this section after the Example one

File: integrations/extensions/starter-kits/metaweather/readme.md (hunk 2 above)
Comment (amiddavid) [reply integrations/extensions/starter-kits/metaweather/readme.md:51]: I think you need to provide here an intro on the fact u can mix & match extensions and then move to the technicalities... at the moment, this feels detached and out of flow

**Review by amiddavid — CHANGES_REQUESTED** [review]

File: integrations/extensions/starter-kits/metaweather/readme.md (hunk 1 above)
Comment (jeesooxkim) [reply integrations/extensions/starter-kits/metaweather/readme.md:50]: Edited to say: "Even More Advanced: Using the MetaWeather Extension with Other Extensions"

File: integrations/extensions/docs/ADVANCED_USAGE.md [hunk 3]
```diff
@@ -117,4 +117,6 @@ For illustration purposes, let's say a customer wanted to be able to link their
 1. User selects which ticket ID to add to Zendesk ticket.
 1. The selected HubSpot ticket information is added into Zendesk ticket as a comment.
 
+For better illustration on how to implement something like this, a MetaWeather and Zendesk skill is included and can be found [here](../starter-kits/metaweather/readme.md#even-more-advanced-using-the-metaweather-extension-with-another-extension) in the starter kit.
```
Comment (amiddavid) [inline integrations/extensions/docs/ADVANCED_USAGE.md:120]: I would vote to remove the hubspot-> zendsesk  hand-waving example and use the example u just added

File: integrations/extensions/starter-kits/metaweather/readme.md [hunk 4]
```diff
@@ -43,12 +43,48 @@ To get started with using the OpenAPI spec, you can upload the `metaweather.adva
...
 ![get-weather](./assets/get-weather.gif)
 
-Free free to contribute to this starter kit, or add other starter kits by following these [contribution guidelines](../../docs/CONTRIBUTING.md).
\ No newline at end of file
+Free free to contribute to this starter kit, or add other starter kits by following these [contribution guidelines](../../docs/CONTRIBUTING.md).
+
+## Even More Advanced: Using the MetaWeather Extension with Another Extension
+Going one step further, extensions are not limited to one per skill -- you can use multiple custom extensions in a skill and even in an action! 
+
+We have published a [Medium blog](https://linktoblog) that showcases this capability. The `metaweather-zendesk.blog.actions.json` found in the `advanced` folder is a two extension example skill that is used in the blog. To configure this spec, you should use the below operations and parameters.
```
Comment (amiddavid) [inline integrations/extensions/starter-kits/metaweather/readme.md:60]:
```suggestion
We have published a [Medium blog](https://linktoblog) that showcases this capability. The `metaweather-zendesk.blog.actions.json` found in the `advanced` folder is a two-extension example skill that is used in the blog. To configure this spec, you should use the below operations and parameters.
```

File: integrations/extensions/docs/ADVANCED_USAGE.md (hunk 3 above)
Comment (jeesooxkim) [reply integrations/extensions/docs/ADVANCED_USAGE.md:120]: You're so right lol

**Pull request merged**